model/src/main/java/org/projectnessie/model/DeltaLakeTable.java

```diff
@@ -37,4 +37,9 @@
 
   @Nullable
   public abstract String getLastCheckpoint();
+
+  @Override
+  public Type getType() {
+    return Type.DELTA_LAKE_TABLE;
```

Comment:
Is it possible to reuse this constant in string form for the `@JsonTypeName` annotation above?